### ISSUE TITLE
Default settings are incomplete

### ISSUE THREAD

**Issue by mikhaelkh** [uBlockOrigin/uBlock-issues#431]
### Prerequisites

- [x] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [x] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [x] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [x] uBlock Origin is the only extension
    - [x] uBlock Origin with default lists/settings
    - [ ] using a new, unmodified browser profile
- [x] I am running the latest version of uBlock Origin
- [x] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description

With a fresh install and with settings reset the `Filter Lists`, `My rules`, `Whitelist` tabs are empty.

### Steps to Reproduce

1. Install uBO dev on Chromium or Reset to default settings

### Expected behavior:

Backup file looks like this:
```
{
  "timeStamp": 1550486884688,
  "version": "1.18.4",
  "userSettings": {
    "advancedUserEnabled": false,
    "alwaysDetachLogger": true,
    "autoUpdate": true,
    "cloudStorageEnabled": false,
    "collapseBlocked": true,
    "colorBlindFriendly": false,
    "contextMenuEnabled": true,
    "dynamicFilteringEnabled": false,
    "externalLists": [],
    "firewallPaneMinimized": true,
    "hyperlinkAuditingDisabled": true,
    "ignoreGenericCosmeticFilters": false,
    "largeMediaSize": 50,
    "parseAllABPHideFilters": true,
    "prefetchingDisabled": true,
    "requestLogMaxEntries": 1000,
    "showIconBadge": true,
    "tooltipsDisabled": false,
    "webrtcIPAddressHidden": false
  },
  "selectedFilterLists": [
    "user-filters",
    "assets.json",
    "public_suffix_list.dat",
    "ublock-resources",
    "ublock-filters",
    "ublock-badware",
    "ublock-privacy",
    "ublock-abuse",
    "ublock-unbreak",
    "easylist",
    "easyprivacy",
    "malware-0",
    "malware-1",
    "plowe-0"
  ],
  "hiddenSettings": {
    "assetFetchTimeout": 30,
    "autoCommentFilterTemplate": "{{date}} {{origin}}",
    "autoUpdateAssetFetchPeriod": 120,
    "autoUpdatePeriod": 7,
    "cacheStorageAPI": "unset",
    "cacheStorageCompression": true,
    "cacheControlForFirefox1376932": "no-cache, no-store, must-revalidate",
    "debugScriptlets": false,
    "disableWebAssembly": false,
    "ignoreRedirectFilters": false,
    "ignoreScriptInjectFilters": false,
    "manualUpdateAssetFetchPeriod": 500,
    "popupFontSize": "unset",
    "requestJournalProcessPeriod": 1000,
    "strictBlockingBypassDuration": 120,
    "suspendTabsUntilReady": false,
    "userResourcesLocation": "unset"
  },
  "netWhitelist": "about-scheme\nchrome-extension-scheme\nchrome-scheme\nmoz-extension-scheme\nopera-scheme\nvivaldi-scheme\nwyciwyg-scheme",
  "dynamicFilteringString": "behind-the-scene * * noop\nbehind-the-scene * inline-script noop\nbehind-the-scene * 1p-script noop\nbehind-the-scene * 3p-script noop\nbehind-the-scene * 3p-frame noop\nbehind-the-scene * image noop\nbehind-the-scene * 3p noop",
  "urlFilteringString": "",
  "hostnameSwitchesString": "no-large-media: behind-the-scene false\nno-scripting: behind-the-scene false",
  "userFilters": ""
}
```

### Actual behavior:
`Filter Lists`, `My rules`, `Whitelist` tabs are empty.
Backup file looks like this:
```
{
  "timeStamp": 1550486041415,
  "version": "1.18.5b4",
  "userSettings": {
    "advancedUserEnabled": false,
    "alwaysDetachLogger": true,
    "autoUpdate": true,
    "cloudStorageEnabled": false,
    "collapseBlocked": true,
    "colorBlindFriendly": false,
    "contextMenuEnabled": true,
    "dynamicFilteringEnabled": false,
    "externalLists": [],
    "firewallPaneMinimized": true,
    "hyperlinkAuditingDisabled": true,
    "ignoreGenericCosmeticFilters": false,
    "largeMediaSize": 50,
    "parseAllABPHideFilters": true,
    "prefetchingDisabled": true,
    "requestLogMaxEntries": 1000,
    "showIconBadge": true,
    "tooltipsDisabled": false,
    "webrtcIPAddressHidden": false
  },
  "selectedFilterLists": [],
  "hiddenSettings": {
    "assetFetchTimeout": 30,
    "autoCommentFilterTemplate": "{{date}} {{origin}}",
    "autoUpdateAssetFetchPeriod": 120,
    "autoUpdatePeriod": 7,
    "cacheStorageAPI": "unset",
    "cacheStorageCompression": true,
    "cacheControlForFirefox1376932": "no-cache, no-store, must-revalidate",
    "consoleLogLevel": "unset",
    "debugScriptlets": false,
    "disableWebAssembly": false,
    "ignoreRedirectFilters": false,
    "ignoreScriptInjectFilters": false,
    "manualUpdateAssetFetchPeriod": 500,
    "popupFontSize": "unset",
    "requestJournalProcessPeriod": 1000,
    "selfieAfter": 11,
    "strictBlockingBypassDuration": 120,
    "suspendTabsUntilReady": false,
    "userResourcesLocation": "unset"
  },
  "netWhitelist": "",
  "dynamicFilteringString": "",
  "urlFilteringString": "",
  "hostnameSwitchesString": "",
  "userFilters": ""
}
```

[What actually happened]

### Your environment

* uBlock Origin version: v1.18.5b4
* Browser Name and version: Chromium 72.0.3626.109 (64-bit)
* Operating System and version: Windows 10 v1607 x64

**Comment by uBlock-user** [comment]
looks like a regression in the latest update..

**Comment by uBlock-user** [comment]
Fixed in [1.18.5b5](https://github.com/gorhill/uBlock/releases/tag/1.18.5b5)
